open-banking-brasil-certificate-standards-1_ID1-ptbr.md

```diff
@@ -266,16 +273,16 @@ stateOrProvinceName = <UF>
 localityName = <Cidade>
 organizationalUnitName = <Código de Participante>
 UID = <Software Statement ID emitido pelo diretório>
-commonName = <FQDN|Wildcard>
+commonName = <FQDN>
 
-[ req_cert_extensions ] 
+[ req_cert_extensions ]
 basicConstraints = CA:FALSE
 subjectAltName = @alt_name
 keyUsage = critical,digitalSignature,keyEncipherment
 extendedKeyUsage = clientAuth
 
-[ alt_name ] 
-DNS = <FQDN|Wildcard>
+[ alt_name ]
+DNS = <FQDN>
```

Comment:
As i mentioned, this would make 4 banks no conformant. This needs to go into an ID2 of the certificate standards not an ID1.

Reply:
Hi GT Security,

In performing a review of https://github.com/OpenBanking-Brasil/specs-seguranca/pull/185/files “DCR - Include examples” a significant normative change was included, I’m assuming that this wasn’t accidental.

This change has an impact to all participants and it is generally considered bad practice to introduce normative substantial changes into an existing ‘implementers draft’. An Implementers draft is a stable document that everyone agrees should be used and adopted as a spec. If you want to change something in a specification or implementers draft that is a normative substantial change then the process is to publish a new draft and agree with the ecosystem when you’re going to migrate to the new ID2.

The immediate impacts of this change as I can see them will be:

Banks that are using certificates with wildcards in the SAN or DN would be non-compliant with the specification, hence why you need a new specification.
The Sandbox directory will need to be updated to prevent sandbox ICP certificates from issuing certificates with wildcards in the SAN or DN -> Change Request.
The production directory will need to be updated to block certificate uploads that have a wildcard in the DN or SAN -> Change request.
The mock bank will need to be updated to reject certificates with wildcards in the SAN or DN.
ICP CA’s will need to be informed (which I assume you’ve done already).
TPPs will need to be given time to purchase and then migrate their certificates which will be a very interesting and potentially issue revealing exercise given Banks problems with DCR that have already emerged.

Ideally a change of this magnitude wouldn’t be included in a ticket entitled “update some examples”. ☺

I’ve included GT Infrastructure for visibility

Kind Regards,
Ralph

Reply:
This point I really don't know @joaorodolfo requested this change... I just bring it to this PR as it could make harder to merge... and I agree with you... I 'm going to to check with him (and Sec wg)

Reply:
Hi guys, the Wildcard won't be a problem for now. For the security perspective it wont't create a issues, but It was to be compliance with CABForum/EV / ICP. 
However, as the @RalphBragg said into private chats, we got an inconsistence scenarios into OBB certificate generated by some Brazilians CAs.

It aren't suppose to use wildcard for EV certicates (OBB Client); but It will be managed by Security WG and we don't need to change noting for now.

Than, if you both are happy, we can "close/resolve this conversation"

Thanks